source/dnode/vnode/src/tsdb/tsdbMemTable.c

```diff
@@ -160,6 +160,7 @@ int32_t tsdbDeleteTableData(STsdb *pTsdb, int64_t version, tb_uid_t suid, tb_uid
     goto _err;
   }
 
+  ASSERT(pPool != NULL);
```

Comment:
It is not implemented to recover from a disk error during vnodeCommit. At this unfortunate time,  it is better to abort with assert of certain invariant, instead of segfault.